pkg/controller/operators/catalog/operator.go

```diff
@@ -39,6 +39,7 @@ import (
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorlister"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/ownerutil"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/queueinformer"
+	schema "github.com/operator-framework/operator-lifecycle-manager/pkg/lib/schema"
```

Comment:
Can we vendor the kube lib for this instead of copying it into our libs? Do we need to update our dependencies to 1.15?

Reply:
Ideally, we would update to 1.15 to take advantage of structural schema feature as even with this carried-over code, I had to strip down couple things due to missing fields in CRD. However, we are still using kube 1.12 for OLM so I wonder if a big jump kube-version-wise will be acceptable. I put up this PR to facilitate the discussion if structural schema should be a hard requirement for CRD update given non-structural schema is still supported.